packer/grapl-service/image.pkr.hcl

```diff
@@ -0,0 +1,163 @@
+packer {
+  required_version = ">= 0.12.0"
+}
```

Comment:
Packer is currently at 1.7.2; I think this block can safely be removed.

Reply:
done, thanks